installer/datafiles/linux.data

```diff
@@ -199,12 +199,15 @@ fi
 # Ensure omsagent is in the omiusers group, but leave omsagent as a group
 /usr/sbin/usermod -g omiusers omsagent 1> /dev/null 2> /dev/null
 
-%Postinstall_200
+%Postinstall_500
```

Comment:
This isn't going to do what you want.  SCOM's postinstall script is completely different from the OMS postinstall script. You don't need this (I'd guess you didn't actually PBUILD this change and try it?) ...

Tell me exactly what it is you're trying to do, and I can try and give guidance on how to do what you want. But please do revert this change.

I suspect your problem is that OMS is getting started, it's because of something happening AFTER this, like perhaps DSC calling the OMS script to start up after it did it's thing?

Note that "enable" ONLY controls what happens at boot time, not at the current time (I think, unless your service script is different than everything else).